test/connection_monitoring/pool-checkin-make-available.json

```diff
@@ -22,7 +22,8 @@
     {
       "type": "ConnectionCheckedOut",
       "connectionId": 1,
-      "address": 42
+      "address": 42,
+      "duration": 42
```

Comment:
Do we need to update the runner to assert on the new duration field or does that code already exist?

Reply:
Yes, we handle it with `self.assertIsNotNone(actual_val)` (found by debugging).